man/bundle-doctor.ronn

```diff
@@ -0,0 +1,18 @@
+bundle-doctor(1) -- Checks for missing OS dependencies
+======================================================
+
+## SYNOPSIS
+
+`bundle doctor` [--quiet]
+
+## DESCRIPTION
+
+Scan the OS dependencies of each of the gems requested in your
+Gemfile(5). If missing OS dependencies are detected, Bundler prints
+them and exists status 1. Otherwise, Bundler prints a success message
+and exists status 0.
+
+## OPTIONS
+
+* `--quiet`:
+  Only output warnings and errors.
```

Comment:
I wasn't sure which options I should add here. Running `bundle help doctor` currently shows `--gemfile`, `--quiet`, `--no-quiet`,  `--no-color`, `--retry=NUM`, `--verbose`, and `--no-verbose`.

It doesn't look like most of the other man pages include anything about the options that are shared between commands, so I left most of them out. I included `quiet` only because there's a specific reference to it in [`Doctor`](https://github.com/bundler/bundler/blob/master/lib/bundler/cli/doctor.rb#L64), but even then it doesn't appear to be a different implementation from other commands.

Let me know how you'd like to handle this!

Reply:
I would document `--quiet` and the `--gemfile`, you can ignore everything else.

Reply:
Sounds good! I borrowed the description of `--gemfile` from [the `bundle install` man page](https://github.com/bundler/bundler/blob/master/man/bundle-install.ronn#L85-L90).